src/instructlab/eval/mt_bench_branch_generator.py

```diff
@@ -56,8 +56,9 @@ def generate(judge_model_name, branch, taxonomy_dir, output_dir):
                 print(f"failed to load {qna_file}. skipping...")
                 continue
             for ex in examples:
-                q, a = ex["question"], ex["answer"]
+                q, a = ex.get("question"), ex.get("answer")
                 if q is None or a is None:
+                    logger.warning(f"Skipping malformed file {qna_file}")
```

Comment:
Just double checking, we aren't going to get these unless there is a problem right?

Reply:
Realized it would have broken before if we were hitting this with a good repo.  And I validated it doesn't happen with a run.  So we should be all good.

Reply:
I think `warning` is the way to go